src/ardoq/azure/api.clj

```diff
@@ -20,7 +20,7 @@
   [client]
   (pprint (doc/ops client)))
 
-(defn doc
+(defn docs
```

Comment:
Rename to `docs` to avoid `doc already refers to: #'clojure.repl/doc in namespace: ardoq.azure.api`